recipe/meta.yaml

```diff
@@ -91,9 +91,9 @@ requirements:
   #optional packages with version dependancies
   run_constrained:
     - tomophantom=2.0.0
-    - astra-toolbox>=1.9.9.dev5,<2.1
-    - tigre=2.4
-    - ccpi-regulariser=22.0.0
+    - astra-toolbox>=1.9.9.dev5,<=2.1
```

Comment:
are we happy with a range of ASTRA versions or do we want to pin to 2.1 only?

Reply:
There's no interface changes between the two so I think it's ok. Backend changes are the user's responsibility.